src/cpp/benders/benders_core/BendersBase.cpp

```diff
@@ -456,6 +468,91 @@ void BendersBase::GetSubproblemCut(SubProblemDataMap& subproblem_data_map)
       shouldParallelize());
 }
 
+namespace
+{
+template<class T>
```

Comment:
Is it appropriate to have this utilitary function in this file ? Could we move it elsewhere ?